docs/internals/contributing/writing-code/unit-tests.txt

```diff
@@ -326,6 +326,13 @@ You can resolve this on Debian-based systems, for example, by running::
     $ apt-get install locales
     $ dpkg-reconfigure locales
 
+You can resolve this for macOS systems by configuring your shell's locale::
```

Comment:
I haven't actually run into this problem before, but I do tend to run tests within a linux VM these days. Are there any particular tests that fail without these languages exported so we can target specific tests when verifying this?